0.4.6 Won't Compile Under Python 2.5

In commit 6a2d69dc1de25d7a623b5cfd835d13bc36eab8d0 a call to the function `PyLong_FromSsize_t` was added, but according to [the documentation](https://docs.python.org/2/c-api/long.html#c.PyLong_FromSsize_t) this function is only available in 2.6 and above.

This leads to the following compile error:
```
greenlet.c: In function ‘green_get_stack_saved’:
greenlet.c:1232:2: warning: implicit declaration of function ‘PyLong_FromSsize_t’ [-Wimplicit-function-declaration]
greenlet.c:1232:2: warning: return makes pointer from integer without a cast [enabled by default]
```

And of course the resulting `ImportError`:
```
python2.5 -c 'import greenlet; print (greenlet, greenlet.__version__)'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: /usr/lib/python2.5/site-packages/greenlet.so: undefined symbol: PyLong_FromSsize_t
```

--- a/greenlet.c
+++ b/greenlet.c
@@ -90,6 +90,10 @@ The running greenlet's stack_start is undefined but not NULL.
 #endif
 #endif
 
+#if PY_VERSION_HEX < 0x02060000
+#define PyLong_FromSsize_t PyInt_FromLong
+#endif
+
 #if PY_VERSION_HEX < 0x02050000
 typedef int Py_ssize_t;
 #endif

